### DISCUSSION TITLE
项目打包提示https://npmmirror.com/mirrors/electron/25.9.3/electron-v25.9.3-win32-x64.zip上下文问题

### DISCUSSION THREAD

**Discussion by ii3lack** [alex8088/electron-vite#333] [Q&A]
![image](https://github.com/alex8088/electron-vite/assets/51691377/46f03482-0817-4322-ad44-5ee9725edffe)
Get "https://npmmirror.com/mirrors/electron/25.9.3/electron-v25.9.3-win32-x64.zip": proxyconnect tcp: dial tcp :0: connectex: The requested address is not valid in its context.
github.com/develar/app-builder/pkg/download.(*Downloader).follow.func1
        /Volumes/data/Documents/app-builder/pkg/download/downloader.go:206
github.com/develar/app-builder/pkg/download.(*Downloader).follow
        /Volumes/data/Documents/app-builder/pkg/download/downloader.go:234
github.com/develar/app-builder/pkg/download.(*Downloader).DownloadNoRetry
        /Volumes/data/Documents/app-builder/pkg/download/downloader.go:128
github.com/develar/app-builder/pkg/download.(*Downloader).Download
        /Volumes/data/Documents/app-builder/pkg/download/downloader.go:112
github.com/develar/app-builder/pkg/electron.(*ElectronDownloader).doDownload
        /Volumes/data/Documents/app-builder/pkg/electron/electronDownloader.go:192
github.com/develar/app-builder/pkg/electron.(*ElectronDownloader).Download
        /Volumes/data/Documents/app-builder/pkg/electron/electronDownloader.go:177
github.com/develar/app-builder/pkg/electron.downloadElectron.func1.1
        /Volumes/data/Documents/app-builder/pkg/electron/electronDownloader.go:73
github.com/develar/app-builder/pkg/util.MapAsyncConcurrency.func2
        /Volumes/data/Documents/app-builder/pkg/util/async.go:68
runtime.goexit
        /usr/local/Cellar/go/1.17/libexec/src/runtime/asm_amd64.s:1581
  ⨯ D:\Code\work\calling-system-fe\calling-queue\node_modules\.pnpm\registry.npmmirror.com+app-builder-bin@4.0.0\node_modules\app-builder-bin\win\x64\app-builder.exe process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
Exit code:
1  failedTask=build stackTrace=Error: D:\Code\work\calling-system-fe\calling-queue\node_modules\.pnpm\registry.npmmirror.com+app-builder-bin@4.0.0\node_modules\app-builder-bin\win\x64\app-builder.exe process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
Exit code:
1

pnpm创建项目，打包提示这个有人碰到吗？

**Comment by alex8088** [comment]
网络问题

**Comment by with-the-winds** [comment]
可以借鉴下这篇文章：https://blog.csdn.net/qq_19297987/article/details/119954579
把打包需要下载的东西都先下载下来存放在本地，这样就不需要在线下载了，亲测可以使用。
